NavigationStack.podspec

```diff
@@ -0,0 +1,21 @@
+Pod::Spec.new do |s|
+  s.name             = 'NavigationStack'
+  s.version          = '0.1.0'
+  s.summary          = 'An alternative SwiftUI NavigationView.'
+
```

Comment:
add this line to avoid warnings:
`s.swift_version = '5.0'`
so it can be pushed to Trunk

Reply:
Done!